src/shady-css/token.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+/**
+ * Class that describes individual tokens as produced by the Tokenizer.
+ */
+class Token {
+  /**
+   * Create a Token instance.
+   * @param {number} type The lexical type of the Token.
+   * @param {number} start The start index of the text corresponding to the
+   * Token in the CSS text.
+   * @param {number} offset The end index of the text corresponding to the Token
+   * in the CSS text.
+   */
+  constructor(type, start, offset) {
+    this.type = type;
+    this.start = start;
+    this.offset = offset;
```

Comment:
I think of start of offset as synonymous, start being short for startOffset. How about `start` and `end`?